src/list-data/conversion-types.ts

```diff
@@ -43,5 +43,25 @@ export const ConversionTypes: ConversionTypesI[] = [
     ],
     value: NrRequestTypeCodes.CONVERT_CORP,
     shortlist: false
+  },
+  {
+    desc: 'Unlimited Liability Company to a limited Company',
+    text: 'Unlimited Liability Company to a limited Company',
+    entity_type_cd: EntityType.CR,
+    blurbs: [
+      'Alter business type from an unlimited Liability Company to a limited Company.'
+    ],
+    value: NrRequestTypeCodes.CONVERT_ULCB,
```

Comment:
See all known NR Request Type codes here:
https://github.com/bcgov/bcrs-shared-components/blob/main/src/enums/name-request-codes.ts

Note that we don't support all of them yet, which was the source of Mihai's concern this morning. He's looking at the new codes in that table and will create tickets for missing functionality.

Reply:
Background/aside: some of those new codes are used for "Societies" (which doesn't use Namerequest UI or LEAR yet, but may be added soon).